w3c/aria-at-app
# Run configuration now shows existing active runs

How to test:
- run the dev server
- go to the new "configure active runs" page
- make some runs
- click "save run configuration" and see the popup saying a save is happening
- go to the run queue, see all the active runs
- go back to the "configure active runs" page and hard refresh, make sure configuration reloads
- Change the configuration, check that the test queue reflects it, and hard refresh to make sure the changes maintain.
